src/main/java/redis/clients/jedis/MultiNodePipelineBase.java

```diff
@@ -159,6 +145,9 @@ public final void sync() {
     } catch (InterruptedException e) {
       log.error("Thread is interrupted during sync.", e);
     }
+
```

Comment:
We don't need to close the thread pool every time we use it up, hope it is a static object and avoid creating it every time. (We don't need a fallback to newFixedThreadPool strategy)